CONTRIBUTING.asciidoc

```diff
@@ -89,19 +89,38 @@ Checkers
 qutebrowser uses http://tox.readthedocs.org/en/latest/[tox] to run its
 unittests and several linters/checkers.
 
-Currently, the following tools will be invoked when you run `tox`:
-
-* Unit tests using https://www.pytest.org[pytest].
-* https://pypi.python.org/pypi/pyflakes[pyflakes] via https://pypi.python.org/pypi/pytest-flakes[pytest-flakes]
-* https://pypi.python.org/pypi/pep8[pep8] via https://pypi.python.org/pypi/pytest-pep8[pytest-pep8]
-* https://pypi.python.org/pypi/mccabe[mccabe] via https://pypi.python.org/pypi/pytest-mccabe[pytest-mccabe]
-* https://github.com/GreenSteam/pep257/[pep257]
-* http://pylint.org/[pylint]
-* https://pypi.python.org/pypi/pyroma/[pyroma]
-* https://github.com/mgedmin/check-manifest[check-manifest]
-* `scripts/misc_checks.py` which checks for the following things:
+Currently, following tox environments are available:
+
+* Unit tests using https://www.pytest.org[pytest]:
+  - `py34`: Run pytest for python-3.4.
+  - `py35`: Run pytest for python-3.5.
+  - `py34-cov`: Run pytest for python-3.4 with code coverage report.
+  - `py35-cov`: Run pytest for python-3.4 with code coverage report.
+  - `unittests-frozen`: ?
+  - `unittests-nodisp`: ?
```

Comment:
Not sure if they should be mentioned at all? If so, `frozen` would be "Run pytest on tests frozen by cx_Freeze (mostly useful on Windows)" and `nodisp` would be "Run tests without `DISPLAY` set (on Linux)" or something.